net-irc/znc/znc-1.7.0.ebuild

```diff
@@ -49,8 +49,13 @@ DEPEND="
 	nls? ( sys-devel/gettext )
 	perl? ( >=dev-lang/swig-3.0.0 )
 	python? ( >=dev-lang/swig-3.0.0 )
+	test? ( >=dev-qt/qtnetwork-5.4 )
```

Comment:
Version 5.3 was removed in https://github.com/gentoo/gentoo-gitmig-20150809-draft/commit/385ea237297058588d6da88ad325082102203575 so I think it's overkill to specify a version range.
You should add a slot though: `dev-qt/qtnetwork:5`